### PR TITLE
Community Portal Event rsvp support

### PR DESCRIPTION
**Pull request by BradHN1**

### CODE REVIEW

File: v2.0/src/AppRouter.js [hunk 1]
```diff
@@ -271,6 +271,7 @@ class AppRouter extends Component {
         apiCall("users.actions.completed.list", { email: user.email }),
         apiCall("users.events.list", { email: user.email }),
       ]);
+      console.log('EVENTS RSVP LIST RESPONSE :)) ', eventsRsvpListResponse);
```
Comment (Opoku-Agyemang) [inline v2.0/src/AppRouter.js:274]: remove

File: v2.0/src/Components/Pages/EventsPage/EventsPageReal.js [hunk 2]
```diff
@@ -31,25 +31,8 @@ class EventsPage extends React.Component {
...
-    .catch((err) => {
-      console.log(err);
-    })
-    /*apiCall('events.exceptions.list', {'event_id': event.id })
-    .then((json) => {
-    })
-    .catch((err) => {
-      console.log(err);
-    });*/
+    console.log('checking if the list of event rsvps is here', this.props);
```
Comment (Opoku-Agyemang) [inline v2.0/src/Components/Pages/EventsPage/EventsPageReal.js:34]: remove

File: v2.0/src/config/config.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "IS_LOCAL": false,
+  "IS_LOCAL": true,
```
Comment (Opoku-Agyemang) [inline v2.0/src/config/config.json:2]: flip?

**Review by Opoku-Agyemang — CHANGES_REQUESTED** [review]

**Review by isha-san — APPROVED** [review]
The event.date.update apiCall should be in AppRouter :)

File: v2.0/src/AppRouter.js (hunk 1 above)
Comment (BradHN1) [reply v2.0/src/AppRouter.js]: done

File: v2.0/src/Components/Pages/EventsPage/EventsPageReal.js (hunk 2 above)
Comment (BradHN1) [reply v2.0/src/Components/Pages/EventsPage/EventsPageReal.js]: done

File: v2.0/src/config/config.json (hunk 3 above)
Comment (BradHN1) [reply v2.0/src/config/config.json:2]: aready done - I think you were looking at local code

**Pull request merged**